src/storage/config.rs

```diff
@@ -24,10 +24,10 @@ const DEFAULT_SCHED_CAPACITY: usize = 10240;
 const DEFAULT_SCHED_MSG_PER_TICK: usize = 1024;
 const DEFAULT_SCHED_CONCURRENCY: usize = 102400;
 
-// According to "Little's law", assuming you can write 100_000 KVs
-// per second, and it takes about 100ms to process the write requests
-// on average, hence using the 10_000 as the default value here.
-const DEFAULT_SCHED_TOO_BUSY_THRESHOLD: usize = 10_000;
+// According to "Little's law", assuming you can write 100MB per
+// second, and it takes about 100ms to process the write requests
+// on average, hence using the 10MB as the default value here.
+const DEFAULT_SCHED_PENDING_WRITE_MB: u64 = 10;
```

Comment:
I think we can tolerate a larger latency (e.g. 1s), so maybe 100MB is more proper?